tests/integration_tests/run_constant_reprocessing_openmc/test.py

```diff
@@ -74,14 +80,17 @@ def read_nuclide_mass(db_file):
     db = tb.open_file(db_file, mode='r')
     fuel_before = db.root.materials.fuel.before_reproc.comp
     fuel_after = db.root.materials.fuel.after_reproc.comp
-    nucmap = fuel_before.attrs.iso_map
+
+    before_nucmap = _create_nuclide_map(db.root.materials.fuel.before_reproc)
+    after_nucmap = _create_nuclide_map(db.root.materials.fuel.after_reproc)
 
     mass_before = {}
     mass_after = {}
 
-    for nuc in nucmap:
-        mass_before[nuc] = np.array([row[nucmap[nuc]] for row in fuel_before])
-        mass_after[nuc] = np.array([row1[nucmap[nuc]] for row1 in fuel_after])
+    for nuc, idx in before_nucmap.items():
+        mass_before[nuc] = np.array([row[idx] for row in fuel_before])
+    for nuc, idx in after_nucmap.items():
+        mass_after[nuc] = np.array([row1[idx] for row1 in fuel_after])
```

Comment:
I see it was done previously, but renaming from `row` to `row1` here seems unnecessary